### ISSUE TITLE
Any plans on adding current method?

### ISSUE THREAD

**Issue by iamsahebgiri** [eidellev/adonis-stardust#5]
I was wondering if stardust is going to implement current method as Ziggy does.

`current()` basically returns the current named routes if no parameters is passed. If any named routes is passed as an arguments it match with the current URL. Here is what stardust api looks might looks like if implemented - 

```js
// Route called 'users.index', with URI '/users'
// Current window URL is https://localhost:3333/users

import { stardust } from '@eidellev/adonis-stardust'

stardust.current()  // 'users.index'
stardust.current('users.index'); // true
```
Ref: https://github.com/tighten/ziggy#the-router-class

This is how they have implemented - 
https://github.com/tighten/ziggy/blob/main/src/js/Router.js#L71-L101

**Comment by eidellev** [comment]
Hey @iamsahebgiri !
Thanks for the feature request. I am currently working on V2 of stardust and I will try to add this to the upcoming release.

**Comment by eidellev** [comment]
Implemented in `v2.1.0`
